lang/py/avro/test/test_script.py

```diff
@@ -193,26 +178,29 @@ def tearDown(self):
                 continue
 
     def _run(self, *args, **kw):
-        args = [sys.executable, SCRIPT, "write", "--schema", self.schema_file] + list(args)
+        args = [sys.executable, "-m" "avro", "write", "--schema", self.schema_file] + list(args)
```

Comment:
Hah, this one is funny, because `python -mavro` works just as well as `python -m avro`, but it was a good find by the code scanner anyway.